lib/src/presentation/managers/home/home_state.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flower_app/src/domain/entities/home/product_model.dart';
+
+import '../../../domain/entities/home/best_seller_model.dart';
+import '../../../domain/entities/home/occasion_model.dart';
+
+sealed class HomeState {
+  const HomeState();
+}
+
+class HomeStateLoading extends HomeState {
+  const HomeStateLoading();
+}
+
+class HomeStateSuccess extends HomeState {
```

Comment:
we can make object called products list contains three lists according to make there list. 
```
class ProductsLists{
 final List<ProductsModel>? products;
  final List<BestSellerModel>? bestSeller;
  final List<OccasionsModel>? occasions;
 // add constructor
  
}